package.json

```diff
@@ -4,6 +4,7 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve app",
+    "prebuild": "node bin/prebuild.js",
```

Comment:
When do we run this command?

Reply:
It runs automatically when you run eg. `npm run build`

Reply:
The should be a message in the console